src/string.c

```diff
@@ -22,13 +24,13 @@ static val_t String_ctor(struct v7 *v7, val_t this_obj, val_t args) {
 
 static val_t Str_fromCharCode(struct v7 *v7, val_t this_obj, val_t args) {
```

Comment:
This is actually our style guide :-)

- Two spaces before comments
- Space after cast

Limiting number of spurious re-formatting would greatly help usefulness of `git blame`.